ingest/importer/importer.py

```diff
@@ -12,7 +12,7 @@
 format = '[%(filename)s:%(lineno)s - %(funcName)20s() ] %(asctime)s - %(name)s - %(levelname)s - %(message)s'
 logging.basicConfig(format=format)
 
-MAX_ROW_LIMIT = 10
+MAX_ROW_LIMIT = 10000
```

Comment:
Can we have this as an environment variable?

Reply:
As I mentioned in the prev comments, there are a number of constants already hardcoded (for e.g. START_DATA_ROW, HEADER_ROW_IDX) at the moment, and I wasn't sure why use an env var for this particular one. I think best thing to do then is use MAX_ROW_LIMIT as default and any env var can override this, though we will soon loose track of all env vars. Not sure if there's a way of doing this using properties file like in Java? I have merged for now, as 10,000 was mentioned as a reasonable number in the meeting.

Reply:
I see. Agreed. I think we should have a config file/script parameters instead of env var to set this kind of info.